config/webpack/webpack.config.js

```diff
@@ -8,21 +14,77 @@ const basedir = join(__dirname, '../..');
 
 const config = require('../config');
 
+const isDevelopment = process.env.NODE_ENV === 'development';
 const isProduction = process.env.NODE_ENV === 'production';
 
+const ifDevelopment = (...items) => {
+  return isDevelopment ? items : [];
+};
+
+const ifProduction = (...items) => {
+  return ifProduction ? items : [];
```

Comment:
Should check `isProduction`, not `ifProduction`?

Reply:
I removed the functions entirely.